lib/license_scout/overrides.rb

```diff
@@ -420,6 +420,8 @@ def default_overrides
         ["xml-simple", "Ruby", ["https://raw.githubusercontent.com/maik/xml-simple/master/README.md"]],
         ["zonefile", "MIT", ["https://raw.githubusercontent.com/boesemar/zonefile/master/LICENSE"]],
         ["sync", "BSD-2-Clause", ["https://raw.githubusercontent.com/ruby/sync/master/LICENSE.txt"]],
+        ["sprockets", nil, [canonical("MIT")]],
```

Comment:
this should perhaps be added after line #338 as MIT-LICENSE

Reply:
could you please also change nil to "MIT"

Reply:
Agree, I have updated.

Reply:
ahh so this does have a file to be pulled so it might be better to add it into the next section.